packages/py-api/py_api/controllers/hackathon_participants_controller.py

```diff
@@ -0,0 +1,69 @@
+import json
+
+from bson.json_util import dumps
+from bson.objectid import ObjectId
+from fastapi.responses import JSONResponse
+from py_api.database.initialize import participants_col
+from py_api.models import RandomParticipant
+from py_api.utilities.parsers import has_prohibited_characters
+
+
+class PartcipantsController:
+
+    @classmethod
+    def get_all_participants(cls) -> JSONResponse:
```

Comment:
We will probably extract this function so that it works for the teams collection as well, but for now, leave it as it is.